### PR TITLE
Load rules from files instead of directly specifying them in the config

### PR DESCRIPTION
**Pull request by Pchelolo**
Follow-up from the offsite discussion

cc @wikimedia/services

### CODE REVIEW

File: rules/restbase_deps.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+summary_rerender:
```
Comment (d00rman) [inline rules/restbase_deps.yaml]: Could collapse summary_rerender and definition_rerender now that we have cases.

File: sys/kafka.js [hunk 2]
```diff
@@ -26,7 +27,16 @@ class Kafka {
             produce_dc: options.produce_dc,
             dc_name: options.dc_name
         });
-        this.staticRules = options.templates || {};
+        this.staticRules = options.rules
```
Comment (d00rman) [inline sys/kafka.js]: To keep ourselves sane, this change should be backwards-compatible.

Comment (Pchelolo) [inline sys/kafka.js]: Why? If we allow mixing inline and file-based rules the config structure would look quite ugly.. I don't really see value in backwards-compatibility except little simplification of deploying this.

Comment (d00rman) [inline sys/kafka.js]: the "except" part being key. It wouldn't be possible to have it both ways. Either you define the list of files, or you use the templates stanza as before, in that order. Whichever is satisfied first, wins.

**Comment by d00rman** [comment]
Hm, we will now need a way to template the rules themselves. Different URIs are used for ORES and RB, e.g., outside and inside production. We need to get them in somehow.

**Comment by Pchelolo** [comment]
@d00rman I've managed to solve the problem in a quite hacky way, but it works. Not sure whether the benefits of moving most of configs out of puppet overweight the hackiness of this change. What do you think?

**Comment by d00rman** [comment]
Hm hm, not bad, but indeed hacky. Let's sleep on it for now, and get the other updates out first.

**Comment by Pchelolo** [comment]
After the config deploy via scap3 was implemented it's not such a pain as it's used to be, abandoning this wacky solution

**Pull request closed without merging**